src/tests/ftest/util/data_mover_utils.py

```diff
@@ -0,0 +1,167 @@
+        # verbose output
+        self.verbose = FormattedParameter("--verbose", False)
+        # quiet output
+        self.quiet = FormattedParameter("--quiet", False)
+        # print help/usage
+        self.print_usage = FormattedParameter("--help", False)
+        # source path
+        self.src_path = BasicParameter(None)
+        # destination path
+        self.dest_path = BasicParameter(None)
+
+    def get_param_names(self):
+        """Overriding the original get_param_names"""
+
+        param_names = super(DataMoverCommand, self).get_param_names()
+
+        # move key=dest_path to the end
+        param_names.sort(key='dest_path'.__eq__)
+
+        return param_names
+
+    def set_datamover_params(self, src_pool=None, dst_pool=None, src_cont=None,
+                             dst_cont=None, display=True):
+        """Set the datamover params for the DAOS group, pool, and cont uuid.
+
+        Args:
+          src_pool(TestPool): source pool object
+          dst_pool(TestPool): destination pool object
+          src_cont(TestContainer): source container object
+          dst_cont(TestContainer): destination container object
+          display (bool, optional): print updated params. Defaults to True.
+        """
+
+        # set the obtained values
+        if src_pool:
+            self.daos_src_pool.update(src_pool.uuid,
+                                      "daos_src_pool" if display else None)
+            # setting src service level
+            src_svcl = ":".join(
+                [str(item) for item in [
+                    int(src_pool.pool.svc.rl_ranks[index])
+                    for index in range(src_pool.pool.svc.rl_nr)]])
+            self.daos_svcl.update(src_svcl, "svcl" if display else None)
+#            self.daos_src_svcl.update(src_svcl, "svcl" if display else None)
+
+        if dst_pool:
+            self.daos_dst_pool.update(dst_pool.uuid,
+                                      "daos_dst_pool" if display else None)
+            # setting destination service level
+#            dst_svcl = ":".join(
+#                [str(item) for item in [
+#                    int(dst_pool.pool.svc.rl_ranks[index])
+#                    for index in range(dst_pool.pool.svc.rl_nr)]])
+#            self.daos_dst_svcl.update(dst_svcl, "svcl" if display else None)
+
+        if src_cont:
+            self.daos_src_cont.update(src_cont.uuid,
+                                      "daos_src_cont" if display else None)
+        if dst_cont:
+            self.daos_dst_cont.update(dst_cont.uuid,
+                                      "daos_dst_cont" if display else None)
+
+
+class DataMover(DataMoverCommand):
+    """Class defining an object of type DataMoverCommand."""
+
+    def __init__(self, hosts, timeout=30):
+        """Create a datamover object."""
+        super(DataMover, self).__init__(
+            "/run/datamover/*",
+            "/home/standan/mpiio/install/mpifileutils/bin/dcp")
```

Comment:
```suggestion
            "dcp")
```
When pointing to the pending artifacts (see comment) this should now be available.

Reply:
Done